exporter/doc/population.go

```diff
@@ -24,15 +24,15 @@ func (p Population) FindStratification(ID string) *Stratification {
 }
 
 type Stratification struct {
-	ID    string
-	Value int
+	ID    string 	`json:"id,omitempty"`
+	Value int 		`json:"value,omitempty"`
 }
 
 type SupplementalData struct {
-	Sex       map[string]int
-	Ethnicity map[string]int
-	Race      map[string]int
-	Payer     map[string]int
+	Sex       map[string]int 	`json:"SEX,omitempty"`
+	Ethnicity map[string]int    `json:"ETHNICITY,omitempty"`
+	Race      map[string]int    `json:"RACE,omitempty"`
+	Payer     map[string]int    `json:"PAYER,omitempty"`
```

Comment:
For whatever reason, Health Data Standards expects these population codes to be in all-caps, which I suspect is why we're doing this here.